### PR TITLE
Re-add install-plugin-deps to CLI

### PR DESCRIPTION
**Pull request by dychen**
### Background

Install plugin deps script is currently a standalone script in the `scripts` directory. This adds it to the CLI.

### Changes

Updated the CLI to include an optional `--install-plugin-deps` parameter that installs plugin dependencies during runtime.

### Documentation

You can run it with either:
```
$ ./run.sh --install-plugin-deps
```
Or:
```
$ python -m autogpt --install-plugin-deps
````

### PR Quality Checklist
- [ ] My pull request is atomic and focuses on a single change.
- [ ] I have thoroughly tested my changes with multiple different prompts.
- [ ] I have considered potential risks and mitigations for my changes.
- [ ] I have documented my changes clearly and comprehensively.
- [ ] I have not snuck in any "extra" small tweaks changes

### CODE REVIEW

**Comment by dychen** [comment]
@ntindle

File: autogpt/plugins.py [hunk 1]
```diff
@@ -18,6 +20,23 @@
 from autogpt.models.base_open_ai_plugin import BaseOpenAIPlugin
 
 
+def install_plugin_dependencies():
```
Comment (ntindle) [inline autogpt/plugins.py]: should this call into the file in scripts or does this replace it?

**Review by ntindle — CHANGES_REQUESTED** [review]

File: autogpt/plugins.py (hunk 1 above)
Comment (dychen) [reply autogpt/plugins.py]: Good call. Didn't realize `scripts` was a module. Updated

**Review by ntindle — DISMISSED** [review]

**Comment by ntindle** [comment]
Run the isort fix and we can merge :)

**Review by ntindle — APPROVED** [review]

**Comment by h4ck3rk3y** [comment]
Hey! @ntindle @dychen  curious when this will be available in an image? I am maintaining an easy way to spin up AutoGPT with different backends and plugins; and wanted to add support for `--instal-plugin-deps` !

**Pull request merged**